src/sema/ast.rs

```diff
@@ -60,6 +60,7 @@ pub enum Type {
     UserType(usize),
     /// There is no way to declare value in Solidity (should there be?)
     Value,
+    #[default]
```

Comment:
I think this is a bit strange. Why is `Void` the default? I am not sure `Type` should have a default

Reply:
It's very useful in tests. Without this, deriving default of other structs containing a `Type` isn't possible.

> Why is Void the default? 

Well, I had to pick one. I also saw `Unreachable` as a candidate, but it doesn't have any doc-comment so I wasn't too sure.